02/app.js

```diff
@@ -15,24 +17,40 @@ registerForm.addEventListener('submit', function(e) {
 
     if(!loginEmail.value.includes('@')) {
         errors.push(loginEmailLabel);
+    } else if(loginEmail.value.includes('@')) {
+        checkAgain(errors, corrected, loginEmailLabel)
```

Comment:
Ta opcja nie jest najlepsza, bo jeśli chcesz sprawdzić ponownie to pojawi się ten sam problem.
Wystarczy na samym początku `checkDate()` "zerować" tablicę tj `errors = []` i po problemie ;)